### ISSUE TITLE
test파일을 만들어 보고 싶은데 팁이 있을까요 ?

### ISSUE THREAD

**Issue by HaloKim** [deepseasw/seq2seq_chatbot#1]
좋은 소스 감사합니다 !
케라스 모델 저장을 이용할 수 있는 방법이 있을까요 ?

from keras.models import load_model
model = load_model('seq2seq.h5')
model.compile(optimizer='rmsprop',
              loss='categorical_crossentropy',
              metrics=['accuracy'])
input_encoder = x_encoder[2].reshape(1, x_encoder[2].shape[0])
input_decoder = x_decoder[2].reshape(1, x_decoder[2].shape[0])
results = model.predict([input_encoder, input_decoder])
indexs = np.argmax(results[0], 1)
sentence = convert_index_to_text(indexs, index_to_word)

위와 같이 테스트를 이용하려고 했는데
결과가 원하는대로 나오지 않고 <PADDING>만 추가되어 나오는데
test파일을 작성하는 팁을 알 수 있을까요 ?

with open('w2i.pkl', 'rb') as f:
    word_to_index = pickle.load(f)
with open('i2w.pkl', 'rb') as f:
    index_to_word = pickle.load(f)

딕셔너리는 모델생성시 나온 pkl을 로드해서 사용하였습니다 !
